Frameworks/Foundation/NSOperation.mm

```diff
@@ -131,139 +128,109 @@ - (void)setThreadPriority:(double)priority {
  @Status Interoperable
 */
 - (void)setCompletionBlock:(void (^)(void))block {
-    id oldBlock = priv->completionBlock;
-    priv->completionBlock = [block copy];
-    [oldBlock release];
+    pthread_mutex_lock(&_completionBlockLock);
+    [_completionBlock autorelease];
```

Comment:
switched to strongId

---
In reply to: [69624034](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69624034) [](ancestors = 69624034,69352043,69351630)